multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -147,12 +147,20 @@ func (c *Cluster) Create(stackBody string) error {
 		return err
 	}
 
+	tags := make([]*cloudformation.Tag, 0)
```

Comment:
get rid of the second 0 argument, it's not necessary if we're using the append approach.